docs/release-instruction.md

```diff
@@ -26,33 +28,36 @@ Use `vX.Y.Z` tag for releases and `vX.Y.Z-rc.N` for release candidates
 following the [semantic versioning](https://semver.org/) standard. Tag must be
 created from the latest commit of the master branch.
 
-## Push changes and release tag to github
+## Push changes and release tag to GitHub
 
 This step should bypass the default PR mechanism to get a correct result (so
 that releasing requires admin privileges for the project), both the `master`
 branch update and tag must be pushed simultaneously like this:
 
     $ git push origin master v0.12.0
 
-## Prepare and push images to a dockerhub
+## Prepare and push images to a Docker Hub
 
-Create images for `neofs-storage`, `neofs-ir` and `neofs-cli` applications
-and push them into dockerhub (so that releasing requires privileges in nspccdev
-organization in dockerhub)
+Create images for `neofs-storage`, `neofs-ir` and `neofs-cli`, `neofs-adm` applications
+and push them into Docker Hub (so that releasing requires privileges in nspccdev
+organization in Docker Hub)
 
-    $ make images
-    $ docker push nspccdev/neofs-storage:0.12.0
-    $ docker push nspccdev/neofs-ir:0.12.0
-    $ docker push nspccdev/neofs-cli:0.12.0
+    $ make images && make image-storage-testnet
```

Comment:
Can we add `image-storage-testnet` into the `images` target deps?

Reply:
sure